cms/models.py

```diff
@@ -772,7 +814,7 @@ def is_course_page(self):
         FieldPanel("video_url"),
     ]
 
-    subpage_types = []
+    subpage_types = ["FlexiblePricingRequestForm", "CertificatePage"]
```

Comment:
This is a sub_type of both `ProgramPage` and `CoursePage` and both of these classes are children of the `ProductPage` class. That's why I moved it to the parent class i.e. `ProductPage` as I've moved the `CertificatePage` here. If you think this can cause some issues I'll move it back.

Reply:
I thnk It's fine then, we just need to be careful about that we don't break flexible pricing related things while working on certificates.

Reply:
I'm reverting it so no uncertainty happens

Reply:
After reviewing it carefully I think it'll not break anything. Also, moving it back to `CoursePage` and `ProgramPage` replaces the `subpage_types` inherited from `ProductPage` that causes some tests to fail on `CoursePage` and `ProgramPage` related to `CertificatePage`, to fix these tests, I'll have to add both `CertificatePage` and `FlexiblePricingRequestForm` as subpage_types on both child classes (`CoursePage` and `ProgramPage`) of `ProductPage`, which will be duplication. So I just left it as it is